test/bindings-test.js

```diff
@@ -3207,6 +3207,50 @@ testHelpers.dev.devOnlyTest('using @viewModel shows a deprecation warning', func
     equal(teardown(), 1);
 });
 
+test("set string on the viewModel", function(){
+	var ViewModel = DefaultMap.extend({
+		foo: {
+			type: "string",
+			set: function(val){
+				equal(val, "bar");
+			}
+		},
+		baz: {
+			type: "string",
+			set: function(val){
+				equal(val, "qux");
+			}
+		}
+	});
+
+	MockComponent.extend({
+		tag: "test-elem",
+		viewModel: ViewModel
+	});
+
+	var template = stache("<test-elem foo:from=\"'bar'\" baz:from=\"'qux'\"/>");
+	template();
+});
+
+test("warn about using attributes to set values on the viewModel", function(){
+	var teardown = testHelpers.dev.willWarn(/foo="bar" is deprecated\. Use foo:from="'bar'" instead\./);
```

Comment:
This tests seems to imply that only one of these attributes gives a warning, when really they both should. Can you update the regex to match both cases?